### PR TITLE
feat(ins): added volumes to sbs migration documentation

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

**Review by ldecarvalho-doc — APPROVED** [review]

File: compute/instances/how-to/migrate-volumes-snapshots-to-sbs.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  validation: 2024-08-06
+  posted: 2024-06-06
+categories:
+  - compute
+---
+
+## Migrating an existing Block Storage volume to Block Storage Low Latency
+
+<Message type="important">
+    This process described in below applies only to Block SSD (`b_ssd`) volumes.
```
Comment (nerda-codes) [inline compute/instances/how-to/migrate-volumes-snapshots-to-sbs.mdx:19]:
```suggestion
    This process described below applies only to Block SSD (`b_ssd`) volumes.
```

File: compute/instances/how-to/migrate-volumes-snapshots-to-sbs.mdx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+   --header 'Content-Type: application/json' \
+   --header 'X-Auth-Token: <SCW_SECRET_KEY>' \
+   --data '{
+       "volume_id": "<SCW_VOLUME_ID>"
+   }'
+   ```
+
+   The response includes the volume and any snapshots created from the volume that will be migrated to Block Storage Low Latency. It also returns a `validation_key`.
+
+2. Confirm the Migration by using the following `curl` command:
```
Comment (nerda-codes) [inline compute/instances/how-to/migrate-volumes-snapshots-to-sbs.mdx:35]:
```suggestion
2. Confirm the migration by using the following `curl` command:
```

File: compute/instances/how-to/migrate-volumes-snapshots-to-sbs.mdx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  h1: How to migrate existing volumes and snapshots to Block Storage Low Latency
+  paragraph: This page explains how to migrate your existing Block Storage volumes and snapshots to Scaleway's Block Storage Low Latency.
+tags: instance snapshot volume b_ssd sbs
+dates:
+  validation: 2024-08-06
+  posted: 2024-06-06
+categories:
+  - compute
+---
+
```
Comment (nerda-codes) [inline compute/instances/how-to/migrate-volumes-snapshots-to-sbs.mdx:15]: I think the requirements part is missing

File: compute/instances/how-to/migrate-volumes-snapshots-to-sbs.mdx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+   }'
+   ```
+
+## Migrating an existing Block Storage snapshot to Block Storage Low Latency
+
+<Message type="important">
+    This process applies to Block SSD (`b_ssd`) or Unified (`unified`) snapshots.
+</Message>
+
+1. Plan the Migration by using the following `curl` command to:
```
Comment (nerda-codes) [inline compute/instances/how-to/migrate-volumes-snapshots-to-sbs.mdx:53]:
```suggestion
1. Plan the migration by using the following `curl` command to:
```

File: compute/instances/how-to/migrate-volumes-snapshots-to-sbs.mdx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+   --header 'Content-Type: application/json' \
+   --header 'X-Auth-Token: <SCW_SECRET_KEY>' \
+   --data '{
+       "snapshot_id": "<SCW_SNAPSHOT_ID>"
+   }'
+   ```
+
+   The response includes the volume and any snapshots created from the snapshot that will be migrated to Block Storage Low Latency. It also returns a `validation_key`.
+
+2. Confirm the Migration using the following `curl` command:
```
Comment (nerda-codes) [inline compute/instances/how-to/migrate-volumes-snapshots-to-sbs.mdx:66]:
```suggestion
2. Confirm the migration using the following `curl` command:
```

File: menu/navigation.json [hunk 6]
```diff
@@ -1041,6 +1041,10 @@
                     "label": "Use the snapshot import/export feature",
                     "slug": "snapshot-import-export-feature"
                   },
+                  {
+                    "label": "Migrate volumes and snapshots to Block Storage Low Latency",
```
Comment (bene2k1) [inline menu/navigation.json:1045]:
```suggestion
                    "label": "Migrate volumes and snapshots to Scaleway SBS",
```

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**